nixos/tests/installer.nix

```diff
@@ -68,160 +67,167 @@ let
                   , testCloneConfig
                   }:
     let
+
+      # FIXME taken from tests/boot.nix - should probably be moved to a common location
+      pythonDict = params: "\n    {\n        ${concatStringsSep ",\n        " (mapAttrsToList (name: param: "\"${name}\": \"${param}\"") params)},\n    }\n";
       iface = if grubVersion == 1 then "ide" else "virtio";
       isEfi = bootLoader == "systemd-boot" || (bootLoader == "grub" && grubUseEfi);
 
-      # FIXME don't duplicate the -enable-kvm etc. flags here yet again!
-      qemuFlags =
-        (if system == "x86_64-linux" then "-m 768 " else "-m 512 ") +
-        (optionalString (system == "x86_64-linux") "-cpu kvm64 ") +
-        (optionalString (system == "aarch64-linux") "-enable-kvm -machine virt,gic-version=host -cpu host ");
+      makeMachineConfig = name: pythonDict ({
+        inherit name;
+        # FIXME don't duplicate the -enable-kvm etc. flags here yet again!
```

Comment:
Do you know what that comment was all about? Couldn't figure it out yesterday, though I did not dive into git blame and the history.

Reply:
no i don't know for sure.
`git blame` just yields this: https://github.com/NixOS/nixpkgs/commit/e58624a877342d1fea39b02a5d6528cd53eefefe which doesn't clear it up any further.

i am assuming this refers to qemu-flags being defined here as well:
https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/qemu-flags.nix

Reply:
(btw, the change can go through without this being resolved, just wanted to point it out as it's being ported as-is.)

Reply:
i double checked. the options set here, are exactly the ones in `qemu-flags.nix` except the ram (`-m`) option.

The bigger picture situation is as follows:
- the vm used initially is set up using [qemu-vm.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/virtualisation/qemu-vm.nix#L121) which respects the options from [qemu-flags.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/qemu-flags.nix) and thus does not use the flags defined in [installer.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/installer.nix#L75)
- only the machines created with `create_machine` in [installer.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/installer.nix#L124) do not use the flags defined in `qemu-flags.nix` as they use a completly different codepath, namely `create_startcommand` from [test-driver.py](https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/test-driver/test-driver.py#L241) which uses `qemu-kvm` only

Reply:
@samueldr what do we wanna do about this?